### PR TITLE
🔀 :: (entry-212) 원서 PDF 미리보기 템플릿 변경

### PR DESCRIPTION
**Pull request by HyunSu1768**
close #entry-212

## Summary by CodeRabbit

- **New Features**
  - Enhanced PDF generation capabilities, incorporating detailed academic performance and attendance data.
  - Improved application form layout and structure, adding new fields for better user information representation.

- **Bug Fixes**
  - Adjusted data types for parent relationships across multiple classes, shifting from structured types to flexible string types, which may improve data handling.

- **Documentation**
  - HTML template for application form updated for better accessibility and clarity.

- **Style**
  - Updated styling for tables and headers in the application form to enhance visual appeal and readability.

### CODE REVIEW

**Comment by HyunSu1768** [comment]
@coderabbitai review

**Comment by HyunSu1768** [comment]
@coderabbitai review

**Comment by HyunSu1768** [comment]
@coderabbitai review

**Review by rudeh2926 — APPROVED** [review]

File: application-domain/src/main/kotlin/hs/kr/equus/application/domain/application/usecase/dto/request/UpdateInformationRequest.kt [hunk 1]
```diff
@@ -11,7 +10,7 @@ data class UpdateInformationRequest(
     val applicantTel: String,
     val parentTel: String,
     val parentName: String,
-    val parentRelation: ParentRelation,
+    val parentRelation: String,
```
[the remark this answers is not in the document]
Comment (HyunSu1768) [reply application-domain/src/main/kotlin/hs/kr/equus/application/domain/application/usecase/dto/request/UpdateInformationRequest.kt:13]: Mother, Father, Guardian 말고도 다른 관계가 있을 수 있어서 String 으로 받게 했습니다.

**Pull request merged**